example/tests/src/FunctionalJavascriptTests/ExampleHomeLinkTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\Tests\example\FunctionalJavascriptTests;
```

Comment:
Based on the pattern in Drupal core, I think this namespace should be...

```suggestion
namespace Drupal\Tests\example\FunctionalJavascript;
```

(^^With the corresponding change to the directory name, obviously.)